default_pages/privacy-policy.md

```diff
@@ -0,0 +1,39 @@
+# Privacy Policy
```

Comment:
We've decided that we don't want to maintain default policies with Saleor at the moment. We only want to give a simple mechanism to allow defining it, but it's the shop owner/operator who is responsible for formulating those policies.